pkgs/development/python-modules/hahomematic/default.nix

```diff
@@ -14,7 +14,7 @@
 
 buildPythonPackage rec {
   pname = "hahomematic";
-  version = "1.8.4";
+  version = "1.8.5";
   format = "pyproject";
 
   disabled = pythonOlder "3.9";
```

Comment:
https://github.com/danielperna84/hahomematic/blob/devel/setup.cfg#L31

Reply:
Looks good to me.

```bash
...nixpkgs]$ nix-build -A python38Packages.hahomematic
error: hahomematic-1.8.4 not supported for interpreter python3.8

...nixpkgs]$ nix-build -A python39Packages.hahomematic
these 25 paths will be fetched (31.22 MiB download, 95.93 MiB unpacked):
  /nix/store/15rny21bdydqyns7ny1cnbgcymriyall-python3.9-aiodns-3.0.0
[...]
```

Reply:
Uh yeah, 12 days ago. I have no recollection of what I saw and thought.